### ISSUE TITLE
WCAG: Distinguishable

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/hourglass#330]
# Summary

## Actions Required
- #243 
- BirkbeckCTP/janeway#3994
- BirkbeckCTP/bau#18

## Scope

This review considered: 
1. Use of colour
2. audio control
3. contrast
4.  resize text
5. images of text

# Audit Result

## Use of colour

Colour has not been used as the only visual mans of conveying information.

## Audio Control

There is no auto-play audio.  The audio tracks that do exist come with standard play/pause/stop and volume controls.

## Contrast

Black text on red failed on contrast.  These are theme colours, so there needs to be further guidance on how to use the theme.

the big issue here seems to be with text that appears over background images, especially as between parallax and different widths of screen, the relative locations of text and background vary.  
example:

![image](https://github.com/BirkbeckCTP/hourglass/assets/5330770/c80c987b-bd4e-4cdb-883b-26992cdcfb31 )

There are often bits of images behind headings, e.g. textured circles.  This needs careful thought as that is part of how the heading looks - but especially on mobile screens the black part of the image may end up behind the back text.  Text colours could defined as a colour paring (foreground and background) so that it brings its own background with it and doesn't end up as black text against a black part of a background image.  

examples:

![Image](https://github.com/BirkbeckCTP/hourglass/assets/5330770/13cba559-7fd3-4240-9039-ba5862a0da7e)

![Image](https://github.com/BirkbeckCTP/hourglass/assets/5330770/9016c2fa-78c4-4d6d-a7a3-bbe1f5afeb99)

**Comment by StephDriver** [comment]
Along similar lines - is this intentional?


![Image](https://github.com/BirkbeckCTP/hourglass/assets/5330770/11b256d6-445d-4b8e-bb41-37cce7be9ba2)
